sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/implementation/EmbeddingsUtils.java

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.openai.implementation;
+
+import com.azure.core.util.BinaryData;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import java.nio.charset.StandardCharsets;
+
+/** This class contains convenience methods and constants for operations related to Embeddings */
+public final class EmbeddingsUtils {
+    private static final ObjectMapper JSON_MAPPER = new ObjectMapper();
+
+    // This method applies to both AOAI and OAI clients
+    public static BinaryData addEncodingFormat(BinaryData inputJson) throws JsonProcessingException {
+        JsonNode jsonNode = JSON_MAPPER.readTree(inputJson.toString());
+        if (jsonNode instanceof ObjectNode) {
+            ObjectNode objectNode = (ObjectNode) jsonNode;
+            objectNode.put("encoding_format", "base64");
```

Comment:
What if the user has already set the `encoding_format`?

Reply:
They can't set the value as the encoding_format will be hide internally.
We will always request `encoding_format` in base64 and return as base64 and then convert the base64 string into List<Float>

Reply:
They can still set the `encoding_format` in their own BinaryData when they are using protocol method directly.

Reply:
Updated